src/main/java/frc/robot/subsystems/Climber/Climber.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.subsystems.Climber;
```

Comment:
Packages should be lowercase.
Also, why is this class duplicated?
I'm not looking at this separately yet, figure out the duplication and address my comments on the other file.